diff --git a/Cslib.lean b/Cslib.lean
index daf39478..0b3ed9bf 100644
--- a/Cslib.lean
+++ b/Cslib.lean
@@ -473,6 +473,7 @@ public import Cslib.Logics.Modal.Tableau.GenericDriver
 public import Cslib.Logics.Modal.Tableau.LoopChecking
 public import Cslib.Logics.Modal.Tableau.LoopInduction
 public import Cslib.Logics.Modal.Tableau.Rules
+public import Cslib.Logics.Modal.Tableau.S5Simplification
 public import Cslib.Logics.Modal.Tableau.Saturation
 public import Cslib.Logics.Modal.Tableau.Soundness
 public import Cslib.Logics.Modal.Tableau.SoundnessStep
